### ISSUE TITLE
steps_ahead not simulating next steps in SimulateNARMAX

### ISSUE THREAD

**Issue by Lalith-Sagar-Devagudi** [wilsonrljr/sysidentpy#88]
https://github.com/wilsonrljr/sysidentpy/blob/440bd926892b506506eaa37d3864cfed99e6a2b1/sysidentpy/simulation/_simulation.py#L333

**bug**
When trying to simulate duffing equation 
```
x_1 = 1.0 x_2
x_2 = -0.15 x_1 + 2.75 x_2 - 1.0 x_2^3
```
by giving steps_ahead = 2 or more, it is still predicting the one step forward for all the given test values. And it cannot simulate with out the steps_ahead parameter and also when steps_ahead = 50 or more. Please see the respective screenshots of the simulations.

```
ident_model = np.array([
      [1001, 1001, 1001],
      [1001,    0,    0],
      [2001,    0,    0],
       ])


s = SimulateNARMAX(basis_function=Polynomial(degree=3), calculate_err=True, estimate_parameter=True, extended_least_squares=True)
yhat_simulate = s.simulate(
    X_train=x_train,
    y_train=y_train,
    X_test=x_test,
    y_test=y_test,
    model_code=ident_model,
    steps_ahead=2)
```

**without steps_ahead**
![wostepahead](https://user-images.githubusercontent.com/40135491/203538366-a5cea718-bfe0-47b4-bc57-e15046cbf301.png)
**steps_ahead = 1**
![wstepahead1](https://user-images.githubusercontent.com/40135491/203538369-71b72e3f-2e04-438a-b015-ef51aeb25c50.png)
**steps_ahead = 2**
![wstepahead2nmor](https://user-images.githubusercontent.com/40135491/203538371-d523ba3f-041b-4b73-a93f-9bcf5198d3e0.png)
**steps_ahead = 50**
![wstepahead50](https://user-images.githubusercontent.com/40135491/203538376-f4722b73-ba63-40fa-aac2-1f3c1856a713.png)
**steps_ahead = 100**
![wstepahead100](https://user-images.githubusercontent.com/40135491/203538380-e40fc9af-d936-4b87-b3b5-53c4b224ea3a.png)

Thank you very much for this very insightful and helpful project.

**Comment by wilsonrljr** [comment]
Hey Lalith! Thank you for creating the issue. The issue is only with the 1 step ahead prediction and fix will be on next update. You can use the solution we talked about on discord in the meantime!

**Comment by wilsonrljr** [comment]
fixed in v0.3.0
